modules/custom/az_ranking/js/az-ranking-focal-point-picker.es6.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * @file
+ * Focal Point Picker for az_image media type.
+ *
+ * Adds a clickable overlay to the focal point picker section in the media edit form.
+ */
+
+((Drupal, once) => {
+  Drupal.behaviors.azRankingFocalPointPicker = {
+    attach: (context) => {
+      // Find the focal point picker wrapper in the media form
+      const pickerWrappers = once(
+        'focal-point-picker',
+        '.focal-point-picker-wrapper',
+        context,
+      );
+
+      pickerWrappers.forEach((wrapper) => {
+        // Find the image
+        const image = wrapper.querySelector('.focal-point-picker-image');
+
+        if (!image) {
+          console.warn('No image found in focal point picker wrapper');
```

Comment:
For any console statements that are necessary to keep, you should be able to just add something like this to satisfy eslint:
```suggestion
          // eslint-disable-next-line no-console
          console.warn('No image found in focal point picker wrapper');
```